app/javascript/controllers/dropdown_menu_controller.js

```diff
@@ -0,0 +1,20 @@
+import { Controller } from "@hotwired/stimulus";
+import {enter, leave, toggle} from 'el-transition';
+
+console.log("hello from dropdown controller");
+
+export default class extends Controller {
+  static targets = ["menu", "button"]
+
+  connect() {
+    // call the enter and leave functions
+  }
+
+  toggleMenu() {
+    if(this.menuTarget.classList.contains('hidden')) {
+        enter(this.menuTarget)
+    } else {
+        leave(this.menuTarget)
+    }
+}
```

Comment:
You can remove the if else statement and just toggle the class "hidden" on click